esp-box/esp-box.c

```diff
@@ -18,47 +18,14 @@
 #include "bsp/esp-box.h"
 #include "esp_lcd_touch_tt21100.h"
 #include "esp_lvgl_port.h"
+#include "bsp_err_check.h"
 
```

Comment:
Could we also increase the default draw buffer size? base on our performance measuremens?

Reply:
I think, that we want to do it in some review before, but here it is: https://github.com/espressif/esp-bsp/pull/101#pullrequestreview-1238794515

Reply:
Ah, OK